packages/select/stories/select.stories.tsx

```diff
@@ -34,182 +34,180 @@ export default {
     clearButtonText: 'clear',
     clearButtonProps: {},
   },
-} as Meta;
+};
 
-export const Default: Story = ({
-  autofill,
-  creatable,
-  disabled,
-  helpMessage,
-  isMulti,
-  isClearable,
-  max,
-  min,
-  raw,
-  required,
-  clearButtonText,
-  clearButtonProps,
-}) => (
-  <FormikResults
-    onSubmit={() => {
-      console.log('submitted');
-    }}
-    initialValues={{
-      select: undefined,
-      autoFill1: '',
-      autoFill2: '',
-    }}
-    validationSchema={
-      isMulti
-        ? multiValueSchema('select', required, min, max, !autofill && !raw)
-        : singleValueSchema('select', required, !autofill && !raw)
-    }
-  >
-    <Row>
-      <Col>
-        {autofill ? (
-          <>
+export const _Default: StoryObj<typeof Select> = {
```

Comment:
looks like Drew has a PR up for the generic selects if y'all want to resolve who is handling what